easybuild/easyconfigs/c/cramtools/cramtools-2.0-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'Tarball'
+
+name = 'cramtools'
+version = '2.0'
+
+homepage = 'https://github.com/enasequence/cramtools/'
+description = """CRAMTools is a set of Java tools and APIs for efficient compression of sequence 
+read data. Although this is intended as a stable version the code is released as
+early access. Parts of the CRAMTools are experimental and may not be supported
+in the future."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/enasequence/%(name)s/archive/']
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [(java, javaver)]
+
+modloadmsg = """To execute cramtools run: java -jar \\${EBROOTCRAMTOOLS}/cramtools-2.0.jar"""
```

Comment:
avoid hardcoding `2.0`, use `%(version)s` instead

and since this is a single-line message, there's no need to use triple quotes

```python
modloadmsg = "To execute cramtools run: java -jar \\${EBROOTCRAMTOOLS}/cramtools-%(version)s.jar"
```